packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1649,7 +1646,7 @@ export type ChatHandlers = {
     onLoadPreviousChatMessages: (messagesToLoad: number) => Promise<boolean>;
     onUpdateMessage: (messageId: string, content: string, options?: {
         metadata?: Record<string, string>;
-        attachedFilesMetadata?: FileMetadata[];
+        attachedFilesMetadata?: AttachmentMetadata[];
```

Comment:
Should the variable name here include 'Files'?

Reply:
in other places we just use metadata

![image](https://github.com/Azure/communication-ui-library/assets/73612854/fbb97120-6e15-42f9-b42d-3c0f54d7833e)

Reply:
Would this only be for files or would inline image also have attachmentMetaData?

Reply:
We can make additional naming changes if we want to, are you suggesting we rename to `attachmentMetadata` ?

Reply:
That would be my suggestion if it does also cover inline image. 'attachmentMetadata' would be more clear. What do you think?

Reply:
I'll go through the interfaces and adjust :)